relentless/simulate/lammps.py

```diff
@@ -321,14 +322,24 @@ def pair_map(sim,pair):
                     f = sim.potentials.pair.force((i,j))[flags]
                     for idx in range(Nr):
                         fw.write('{idx} {r} {u} {f}\n'.format(idx=idx+1,r=r[idx],u=u[idx],f=f[idx]))
+                        # find the largest r with a nonzero potential
+                        if abs(u[idx])>0.0 and u[idx+1]==0.0:
+                            rcut_val = r[idx]
```

Comment:
This will sort of work, but it is inefficient and has some bad corner cases. For example, `rcut_val` will be undefined if this condition is never met. I would suggest instead doing this outside the loop with numpy. Something like this might work:
```py
# find r where potential and force are both zero
nonzero_r = numpy.flatnonzero(numpy.logical_and(~np.isclose(u,0), ~np.isclose(f,0)))
if len(nonzero_r) > 1:
    # cutoff at last nonzero r, must not be first r or LAMMPS will raise error
    rcut[(i,j)] = r[nonzero_r[-1]]
else:
    # 0 or 1 points are nonzero, cutoff at second r point
    rcut[(i,j)] = r[1]
```